pkgs/development/libraries/qt-5/modules/qtbase.nix

```diff
@@ -15,6 +15,7 @@
   # optional dependencies
 , cups ? null, postgresql ? null
 , withGtk3 ? false, dconf, gtk3
+, qttranslations ? null
```

Comment:
Is this ? null needed?

Reply:
I think it will break baseScope but I moved things around a bit since the last time I tried. I'll have to double check.